src/Shipment/Concern/EncodesRecipient.php

```diff
@@ -18,6 +22,23 @@ private function encodeRecipient(?ContactDetails $recipient): ?array
             return null;
         }
 
+        $street = $recipient->street;
+        $streetAdditionalInfo = $recipient->streetAdditionalInfo;
+
+        // Remove overlap if extra info starts with the end of street
+        [$street, $streetAdditionalInfo] = $this->removeStreetOverlap($street, $streetAdditionalInfo);
+
+        // Always truncate street to max length
+        [$street, $overflow] = $this->truncateStreet($street, self::$maxStreetLength);
+
+        // Add overflow to extra field only if it is empty
+        if ($overflow && empty($streetAdditionalInfo)) {
+            $streetAdditionalInfo = $overflow;
```

Comment:
Als additionalinfo leeg is kan bv ‘Extra lange straatnaam 3’ verbasterd worden tot ‘Extra lange straatnaa 3 m‘, of zie ik dat verkeerd? Los daarvan, hoe kunnen we beslissen de overflow weg te gooien afhankelijk van iets anders?